core/src/rest/pretty_error.ts

```diff
@@ -0,0 +1,25 @@
+export interface SimplifiedError {
+  [name: string]: string[];
+}
+
+// It's a deeply nested object just let it be like this
+export function prettyError(errors: any): SimplifiedError {
+  const res: SimplifiedError = {};
+  function fmt(obj: any, acum: string = ""): void {
```

Comment:
Nit: Maybe move the function out of the other scope and pass the `res` as parameter

Reply:
ask @DjDeveloperr not me since it was made by him